### PR TITLE
fix: Fix field icon border radius

### PR DESCRIPTION
**Pull request by tdurnford**
## Description
Currently, fields and field icons have a border radius of 2px which creates an awkward space between the two. This PR updates the field and field icons' border and border radius to appear more continuous. 

## Task Item
#minor

### Current
e24b7d67-ce90-4503-983f-f95b757519bb

### Updated
1a76a189-36c1-4b90-8d6b-39b6ee6d9953

### CODE REVIEW

File: Composer/packages/adaptive-form/src/components/fields/OneOfField/OneOfField.tsx [hunk 1]
```diff
@@ -121,6 +121,7 @@ const OneOfField: React.FC<FieldProps> = (props) => {
       <Field
         key={selectedSchema.type}
         expression={expression}
+        hasIcon={options.length > 1}
```
[the remark this answers is not in the document]
Comment (tdurnford) [reply Composer/packages/adaptive-form/src/components/fields/OneOfField/OneOfField.tsx]: It should be options.length > 1 || !isNested

**Review by beyackle — DISMISSED** [review]

File: Composer/packages/adaptive-form/src/components/fields/OneOfField/OneOfField.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hatpick) [reply Composer/packages/adaptive-form/src/components/fields/OneOfField/OneOfField.tsx]: I thought you only want to do this when those icons are clickable, I'm confused

File: Composer/packages/adaptive-form/src/components/fields/StringField.tsx [hunk 2]
```diff
@@ -89,7 +95,7 @@ export const StringField: React.FC<FieldProps<string>> = function StringField(pr
         placeholder={placeholder}
         readOnly={readonly}
         styles={{
-          ...borderStyles(Boolean(transparentBorder), Boolean(error)),
+          ...borderStyles(Boolean(transparentBorder), Boolean(error), hasIcon),
```
Comment (hatpick) [inline Composer/packages/adaptive-form/src/components/fields/StringField.tsx:98]: !!

Comment (tdurnford) [reply Composer/packages/adaptive-form/src/components/fields/StringField.tsx:98]: ??

Comment (hatpick) [reply Composer/packages/adaptive-form/src/components/fields/StringField.tsx:98]: !!hasIcon

Comment (tdurnford) [reply Composer/packages/adaptive-form/src/components/fields/StringField.tsx:98]: I don't know what difference that makes but ok

File: Composer/packages/adaptive-form/src/components/fields/OneOfField/OneOfField.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (tdurnford) [reply Composer/packages/adaptive-form/src/components/fields/OneOfField/OneOfField.tsx]: This fixes the borders for all fields that have icons next to them - clickable and non-clickable.

**Review by hatpick — DISMISSED** [review]

**Review by hatpick — APPROVED** [review]

**Pull request merged**